src/instruments/instrumentstypes.h

```diff
@@ -161,7 +162,14 @@ struct Instrument
     QString abbreviature() const { return !shortNames.isEmpty() ? shortNames.first().name() : QString(); }
 };
 
-using InstrumentList = QList<Instrument>;
+struct PartInstrument {
+    bool part;
```

Comment:
I don't quite get the meaning of the flag "part". Can you please explain it to me?

Reply:
Apparently a wrong name then :wink: 
The Instrument Form returns a list of `PartInstrument` structure, representing the new list, and order, of `Parts`. For processing it is important whether an element of this list represents a newly added `Part` or an already existing `Part`.
E.g. when setting part Soloist, for new parts the flag can be set straight away, for existing `Parts` the undo/redo has to be used.

Reply:
Name wasn't indeed not that clear, changed into `isExistingPart`.